docs/11_view.md

```diff
@@ -1,18 +1,18 @@
-# View Dandisets
+# Viewing Dandisets
```

Comment:
Is there a specific reason why this does not use the infinitive form like the others?

Reply:
all the main headings I had as gerunds, since you aren't "telling" someone to do these actions (Viewing, Publishing, etc.), they encompass a group of task instructions such as "Browse Dandisets"